docs/source/install/kernel_install.rst

```diff
@@ -0,0 +1,16 @@
+.. _kernel_install:
+
+Kernel Installation
+-------------------
+
+IPython can be installed (different python versions, virtualenv or conda environments) can
+be installed as a kernel by following these steps:
+
+* make sure that the desired python installation is active (e.g. activate the environment)
+  and ipython is installed
+* run once ``ipython kernelspec install-self --user`` (or ``ipython2 ...`` or ``ipython3 ...``
+  if you want to install specific python versions)
+
+The last command installs a kernel spec file for the current python installation in
+``~/.jupyter/kernels/``. Kernel spec files are JSON files, which can be viewed and changed with a
+normal text editor.
```

Comment:
This can link to [this section](http://jupyter-client.readthedocs.org/en/latest/kernels.html#kernel-specs) using Intersphinx.